client/src/modules/locations/sector/modal/createUpdate.html

```diff
@@ -0,0 +1,60 @@
+<form name="sectorForm" bh-submit="ModalCtrl.submit(sectorForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li>
+        <span translate>SECTOR.TITLE</span>
+
+        <label class="label label-warning text-uppercase" translate>
+          {{ModalCtrl.action}}
+        </label>
+      </li>
+    </ol>
+  </div>
+  <div class="modal-body">
+    <div class="row">
+      <div class="col-xs-12">
+      <div class="form-group" ng-class="{ 'has-error' : sectorForm.$submitted && sectorForm.country.$invalid }">
+        <label class="control-label" translate> FORM.LABELS.COUNTRY </label>
+        <select class="form-control" name="country" 
+          ng-model="ModalCtrl.sector.country_uuid" id="country"
+          ng-change="ModalCtrl.loadProvinces()"
+          ng-options="c.uuid as c.name for c in ModalCtrl.countries | orderBy:'name'" required>
+          <option value="" disabled translate> {{ ModalCtrl.messages.country }} </option>
```

Comment:
Does `{{ ModalCtrl.messages.country }}` exist?  I only get a blank text on most of these:

![Needs some text](https://user-images.githubusercontent.com/896472/63577432-48d94980-c586-11e9-8cc7-2ac32b7db2d3.PNG)